src/main/cms/cms_menu_imu.c

```diff
@@ -88,6 +88,15 @@ static const char * const osdTableGyroToUse[] = {
 };
 #endif
 
+#ifdef USE_ITERM_RELAX
+static const char * const osdTableItermRelax[] = {
+    "OFF", "RP", "RPY", "RP_INC", "RPY_INC"
+};
+static const char * const osdTableITermRelaxType[] = {
+    "GYRO", "SETPOINT"
+};
```

Comment:
Rather than redefining the lookup table reuse the existing ones created in `cli/settings.c`.

`lookupTableItermRelax`
`lookupTableItermRelaxType`

This will save flash space and also keep a single maintenance point.

Reply:
Fixed